cli/README.md

```diff
@@ -99,51 +99,6 @@ Example to get the number of feeds of a given user:
 
 # Install and updates
 
-## Using git
+For installation using git checkout out the [README](https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#).
```

Comment:
The link is not a README anymore, is it?

Reply:
Whoops, Copy & Paste error - totally wrong URL, good catch!